FSJ_django20_project/FSJ/templates/FSJ/application_list.html

```diff
@@ -0,0 +1,24 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+        {{ award.award_name }}
+{% endblock %}
+
+{% block content %}
+<div>
```

Comment:
Considering indenting so that the content inside block content are all at a lower level than the block itself.